vtex/faststore
# Feat: Adds `EmptyState` component

## What's the purpose of this pull request?

`fs/components`
- [x] Moves `EmptyState` component
- [x] Adds `EmptyState ` doc page

`fs/ui`
- [x] Adds and reviews `EmptyState` styles

`fs/core`
- [x] Review `EmptyGallery` and `EmptyCart` structure (replace)
- [x] Delete  old `EmptyState`'s  related files (storybook and style)
- [x] Replaces temp Loading pages layout


## How to test it?

1. Check the `EmptyState` component using the doc preview link:
 - [EmptyState](https://faststore-site-git-feat-emtpy-state-fs-977-faststore.vercel.app/components/organisms/empty-state)

2. run `yarn dev` on `faststore/core` application and check the `EmptyState` occurrences on:
   a. Open the `CartSidebar` when the cart has no items. 
   b. Search for `Gallery` using the search input.
   c. Try the `Not Found: 404` page, ex: try http://localhost:3000/echo-smart-speaker-99988214/p
   
![image](https://user-images.githubusercontent.com/3356699/231902621-81754dc2-a2e3-4f70-9cf9-c6bbeb9cc482.png)

   

### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/48
